src/utils/safeHref.ts

```diff
@@ -21,33 +29,32 @@ export function configureSafeHref(configObject: SafeHrefConfigObj) {
     if (configObject.origin) {
         globalOrigin = configObject.origin
     }
+
+    if (configObject.additionalProtocols && configObject.additionalProtocols.length) {
+        safeProtocols.push(...configObject.additionalProtocols)
```

Comment:
Are we ok with this mutating the original array?

Reply:
Thought about this, and decided it might be best to not. Changed that :)